assets/src/edit-story/components/panels/textStyle/color.js

```diff
@@ -74,6 +89,31 @@ function ColorControls({ selectedElements, onSetProperties }) {
           label={__('Background color', 'web-stories')}
         />
       </Row>
+      {backgroundColor && (
+        <Row>
+          <Space flex="1" />
+          <ToggleButton
+            icon={<FillIcon width={32} height={32} />}
+            value={state.backgroundType === 'fill'}
+            isMultiple={false}
```

Comment:
What happens if multiple text fields with different settings are selected?

Reply:
Not sure yet, what do you suggest based on past elements? Maybe @samitron7 can bring some clarification on this behavior.

Reply:
Elsewhere our behavior is to show "(multiple)", but how to do that for a toggle like this. It's kind of like selecting two text fields, one of them bold, the other not.

I guess the common behavior in such a case is to mark neither button as selected (as the state is MULTIPLE_STATE and not any of the valid options), and when clicking any background fill button, make all the textfields sync'ed.